llama_stack/providers/remote/inference/fireworks/config.py

```diff
@@ -22,7 +22,7 @@ class FireworksImplConfig(BaseModel):
     )
 
     @classmethod
-    def sample_run_config(cls) -> Dict[str, Any]:
+    def sample_run_config(cls, __distro_dir__: str) -> Dict[str, Any]:
```

Comment:
Needed otherwise:

<img width="1161" alt="Screenshot 2024-12-19 at 15 07 20" src="https://github.com/user-attachments/assets/02d2e7c1-abbc-4151-851a-1bfac3076e70" />